src/actions/config.js

```diff
@@ -82,6 +83,25 @@ export function configDidLoad(config) {
   };
 }
 
+const URL_BASE = 'https://raw.githubusercontent.com/netlify/netlify-cms/master/src/'
```

Comment:
We could also allow i18n objects to be added to the registry, and maintain a list of translations. Then users can grab the strings from wherever and register them how they like, without being dependent on releases.